src/api/lib/options.js

```diff
@@ -95,7 +95,7 @@ export const input = (params) => [
     'Ungarn-Trick: ',
     h('select', { name: 'trick', id: 'trick' }, [
       optionHTML(0, '--', params.trick === 0),
-      optionHTML(1, 'Ungarn als Start', params.trick === 1),
+      optionHTML(1, 'Ungarn als Start', !params.trick || params.trick === 1),
```

Comment:
Ein Problem, was hier auftritt: Wenn ich eine Suche starte und dann auf "Anfrage ändern" klicke, wird es immer wieder auf den Default-Wert gesetzt, unabhängig davon, was ich zuvor in der Suche ausgewählt hatte.

Reply:
Ich konnte es gerade leider nicht testen, weil gar keine Suchanfragen mehr funktionieren.
Ich hab es zu `params.trick === undefined` geändert, um Logikfehler mit Negation und den Zahlen zu vermeiden.

Reply:
Ich habe gerade noch eine Änderung vorgenommen, sodass der Wert "0" korrekt in die Params übernommen wird. Zusammen mit den anderen Änderungen sollte es jetzt passen.